docs/usage/configuration-options.md

```diff
@@ -1834,11 +1834,10 @@ For example, GitHub might automerge a Renovate branch even if it's behind the ba
 
 ## platformCommit
 
-Supports only GitHub App mode and not when using Personal Access Tokens.
+Only use this option if you run Renovate as a [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps).
+It does not apply when you use a Personal Access Token as credential.
 
-To avoid errors, `gitAuthor` or `gitIgnoredAuthors` should be manually adjusted accordingly.
-
-The primary reason to use this option is because commits will then be signed automatically if authenticating as an app.
+You can use `platformCommit` to automatically sign the commits that Renovate creates.
```

Comment:
```suggestion
When `platformCommit` is enabled, Renovate will create commits using GitHub's API (instead of using `git` directly) to utilize GitHub's [Commit signing support for bots and other GitHub Apps](https://github.blog/2019-08-15-commit-signing-support-for-bots-and-other-github-apps/) feature.
```

I think we need to explain the "why" part better.